### ISSUE TITLE
Internal server error (500) for `environment: null` in request

### ISSUE THREAD

**Issue by mdklatt** [mdklatt/httpexec#12]
Passing `environment: null` as part of the POST request results in an internal server error. The problem is in `asgi.run()`:

```python
@app.route("/<path:command>", methods=["POST"])
async def run(command: str):

    # <snip>

    # This fails for environment=None, because that will be passed to _exec() instead of an empty dict.
    env = params.get("environment", {})
    result = await _exec(argv, streams, env)

   # <snip>
```

An explicit value of `null` should be handled the same as if `environment` is missing. As a workaround, clients must ensure that they do not pass an explicit `null` for `environment`.

**Comment by mdklatt** [comment]
Commit 1da489653745ed1bf0a996740d85a69a23472214 is passing tests, but this needs downstream validation.

**Comment by mdklatt** [comment]
Issue resolved by commit 14e321724d7638f79d5c464fc465128099972710.
